azurerm/appserviceplan.go

```diff
@@ -0,0 +1,18 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func appServicePlanStateRefreshFunc(client *ArmClient, resourceGroupName string, appserviceplan string) resource.StateRefreshFunc {
```

Comment:
_really minor_ it'd be good to call `appserviceplan` -> `name` for consistency across these methods, if possible